docs/quickstart.rst

```diff
@@ -0,0 +1,385 @@
+
+    exs = ["Experiment 1 Accession Code", ex2]
+
+You can also pass in other optional parameters to alter the Dataset itself and to alter how the download process works.
+
+* `aggregation` can be used to change how the Dataset is aggregated.
+
+* `transformation` can be used to change the transformation of the Dataset.
+
+* `skip_quantile_normalization` can be used to choose whether or not quantile normalization is skipped.
+
+* `extract` can be used to choose whether the downloaded zip file should be automatically extracted.
+
+* `prompt` can be used to choose whether or not you should be prompted before downloading if the Dataset zip file is larger than 1 gigabyte.
+
+Below is a simple example of downloading a Dataset using `dataset_dict`:
+
+.. code-block:: python
+
+    pyrefinebio.download_Dataset(
+        "./path/to/dataset.zip",
+        "foo@bar.com",
+        dataset_dict={
+            "GSE24528": [
+                "GSM604796",
+                "GSM604797"
+            ]
+        }
+    )
+
+Below is a simple example of downloading a Dataset using `experiments`:
+
+.. code-block:: python
+
+    pyrefinebio.download_Dataset(
+        "~/path/to/dir/for/dataset/",
+        "foo@bar.com",
+        experiments=["GSE24528", "GSE30631"]
+    )
+
+Downloading Compendia
+---------------------
+
+You can start downloading Compendia after you set up and activate a Token.
+
+pyrefinebio provides the function `download_compendium()` that makes downloading Compendium results super easy.
+It will will automatically search for Compendia based on organisms and download the results.
+
+`download_compendium()` requires that you pass in the parameters `path` and `organism`. 
+
+* `path` is the path that the Compendium will be downloaded to. This can be a path to a directory, in which case a filename will automatically be created, or a path to a zip file. 
+
+* `organism` is the name of the Organism for the Compendium that you want to download.
+
+You can also pass in other optional parameters to alter the type of Compendium you download.
+
+* `version` is the Compendium version. The default is `None` which will get the latest version.
+
+* `quant_sf_only` can be used to choose if the Compendium is quantile normalized. Pass in True for RNA-seq Sample Compendium results or False for quantile normalized
+
+* `extract` can be used to choose whether the downloaded zip file should be automatically extracted.
+
+* `prompt` can be used to choose whether or not you should be prompted before downloading if the Compendium zip file is larger than 1 gigabyte.
+
+Below is a simple example of Downloading a Compendium result:
+
+.. code-block:: python
+
+    pyrefinebio.download_compendium(
+        "./path/to/compendium.zip",
+        "DANIO_RERIO",
+    )
+
+pyrefinebio also provides the function `download_quandfile_compendium()` which is equivalent to calling `download_compendium(quant_sf_only=True)`.
+
```

Comment:
This feels like an advanced usage thing. I'm not sure if this should be in quick start. @kurtwheeler what do you think?